Exiled.API/Features/Scp096.cs

```diff
@@ -16,5 +18,10 @@ public class Scp096
         /// Gets or Sets a value indicating the max shield amount Scp096 can have.
         /// </summary>
         public static float MaxShield { get; set; } = 500;
+
+        /// <summary>
+        /// Gets a list of player ids who will be turned away from SCP-096.
+        /// </summary>
+        public static List<int> TurnedPlayers { get; private set; } = new List<int>();
```

Comment:
Since the ghostmode patch already uses player objects for 173's shouldRotate stuff, why is this a List<int> instead of a List<Player> ?

Reply:
A List instead of a List? I don't understand what you're saying.

Reply:
Thanks github very cool..

I meant to ask, why is it a List int 
instead of a List Player

Reply:
I figured I'd make it a `List<int>` since `TargetGhosts` was also an int. I'll change them both though.